### ISSUE TITLE
(Master) Error reverting to database version 274 from 298

### ISSUE THREAD

**Issue by deltecent** [ElvishArtisan/rivendell#289]
```
$ sudo rddbmgr --modify --set-schema=274
DB connection re-established
invalid SQL or failed DB connection[Duplicate column name 'INSTANCE' QMYSQL: Unable to execute query]: alter table RDAIRPLAY add column INSTANCE int unsigned not null default 0 after STATION
rddbmgr: sql error: Duplicate column name 'INSTANCE' QMYSQL: Unable to execute query query: alter table RDAIRPLAY add column INSTANCE int unsigned not null default 0 after STATION
```

**Comment by ElvishArtisan** [comment]
Unable to reproduce this here. Are you sure that you've merged 9338a14042f0ea810a17eeb31aa6921309325e94?

**Comment by deltecent** [comment]
No. Sorry.
